packages/manager/modules/pci/src/projects/project/kubernetes/details/service/service.routing.js

```diff
@@ -18,6 +18,13 @@ export default /* @ngInject */ ($stateProvider) => {
           projectId,
         ) => () => $state.go('pci.projects.project.kubernetes.details.service.name', { kubeId, projectId }),
 
+        clusterMinorVersion: /* @ngInject */ (
```

Comment:
It's more a `version` than a `minorVersion` right ?

Reply:
Actually, it is the version without the security patch part. The complete version for example is `1.11.3-5` and the minor version is `1.11`